test_add_valid (__main__.TestMatrix) ... FAIL
test_get_valid (__main__.TestMatrix) ... ok
test_init_invalid (__main__.TestMatrix) ... ok
test_init_valid (__main__.TestMatrix) ... ok
test_mul_valid (__main__.TestMatrix) ... FAIL
test_properties (__main__.TestMatrix) ... ok
test_set_valid (__main__.TestMatrix) ... ok

======================================================================
FAIL: test_add_valid (__main__.TestMatrix)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\youne\Documents\GitHub\c03-ch11-3-exercices-Younes51-1\test_exercice.py", line 95, in test_add_valid
    self.assertEqual(output, expected)
AssertionError: Matrix(2, 3, [1111, 1212, 1313, 2121, 2222, 2323]) != Matrix(2, 3, [1111, 1212, 1313, 2121, 2222, 2323])

======================================================================
FAIL: test_mul_valid (__main__.TestMatrix)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\youne\Documents\GitHub\c03-ch11-3-exercices-Younes51-1\test_exercice.py", line 105, in test_mul_valid
    self.assertEqual(output, expected)
AssertionError: Matrix(2, 2, [776.0, 812.0, 1406.0, 1472.0]) != Matrix(2, 2, [776, 812, 1406, 1472])

----------------------------------------------------------------------
Ran 7 tests in 0.001s

FAILED (failures=2)
